### ISSUE TITLE
Link the WFO page to the WFO's AFD page

### ISSUE THREAD

**Issue by greg-does-weather** [weather-gov/weather.gov#1568]
### Description

When the AFD tickets (#1526, #1528, #1527, #1529, #1530) are done, we need to update the WFO page so it links to the WFO's AFD page.

### Questions and Considerations

Where should the AFD link take people? We probably can't answer that completely until the AFD page is built, but we'll need to answer it before we can pick this up.

### Acceptance Criteria
- [ ] You can click the link and go where you think you're going

**Comment by colinmurphy01** [comment]
Duplicate of #1617

**Comment by greg-does-weather** [comment]
Not quite a duplicate!
